src/cycax/cycad/cycad_part.py

```diff
@@ -461,6 +461,9 @@ def render(self, engine: str = "Preview3D", engine_config: Optional[dict] = None
             part_engine = PartEngineOpenSCAD(name=self.part_no, path=self._base_path, config={"stl": False})
 
         elif _eng_lower == "freecad":
+            if engine_config is None:
+                engine_config={'out_formats':"PNG: ALL, STL: None, DXF: TOP"}
```

Comment:
I'd expect a more structured and clear format presented to the user. The value of 'out_formats' should be a list. 
Maybe:
```
engine_config['out_formats'] = [('png', 'ALL'), ('STL',), ('DXF',TOP)]
```